decentraland/screens/BuyLandScreen/BuyLandScreen.js

```diff
@@ -119,16 +138,19 @@ export class BuyLandScreen extends React.Component {
         type === ESTATE ? await estateContract.getFingerprint(assetId) : "0x";
 
       marketplaceContract.setAccount(account);
+
+      // TODO: Add extra param back in to reproduce the error
+      // state that we should handle better
       const action = marketplaceContract.safeExecuteOrder(
         nftAddress,
         `${assetId}`,
         `${priceInWei}`,
-        `${fingerprint}`,
-        gasParams
```

Comment:
@marcelomorgado This was the issue that led us down an error path we hadn't traversed before

Reply:
Resolving this since you gave it a thumbs up